app/features/authentication.feature

```diff
@@ -30,13 +30,13 @@ Feature: Authentication
         | email        | shep_clark@example.org|
       And I request "/user" be created
       And a record exists with an employee_id of "123"
-      And the user has a password of "govols!!!"
+      And the user has a password of "89ao7kah09o8u7akh098eu7hka089eu7"
```

Comment:
Would it be better to replace this with a random long password so it is unlikely to ever be pwned?